boards/thingy52/board.c

```diff
@@ -36,4 +37,34 @@ void board_init(void)
 
     /* initialize the CPU */
     cpu_init();
+
+#ifdef MODULE_PWM_DAC
```

Comment:
I think should go to the auto_init part, similar to other drivers. That would make it portable to other boards without having this specific handling.

Reply:
Now auto-initializable.